src/main/kotlin/camp/nextstep/edu/step/step2/domain/lotto/Lottos.kt

```diff
@@ -10,6 +11,17 @@ class Lottos(
         require(lottos.isNotEmpty()) { "로또는 1개 이상이어야 합니다." }
     }
 
+    fun checkLottoNumbersByWinningLotto(winningLotto: WinningLotto): List<LottoMatch> {
+        val lottoMatchResults = mutableListOf<LottoMatch>()
+
+        for (lotto in lottos) {
+            val matchNumbers = lotto.countMatch(winningLotto.winningLotto)
+            val matchBonus = lotto.countMatchBonus(winningLotto.bonusNumber)
+            lottoMatchResults.add(LottoMatch.of(matchNumbers, matchBonus))
+        }
```

Comment:
```suggestion
lottos.map { 
    val matchNumbers = lotto.countMatch(winningLotto.winningLotto)
    val matchBonus = lotto.countMatchBonus(winningLotto.bonusNumber) 
    LottoMatch.of(matchNumbers, matchBonus)
}
```
`val lottoMatchResults = mutableListOf<LottoMatch>()` 선언을 없앨수 있을것 같아요 ~

Reply:
말씀해주신 내용으로 한번 다시 선언해보겠습니다!!